### PR TITLE
#490 changes

### PR DESCRIPTION
**Pull request by mohitsengarr**
Changes for https://github.com/k8-proxy/cs-k8s-api/issues/490

### CODE REVIEW

File: Source/Test/Glasswall.CS-SDK.UnitTests/AnalyseController_AnalyseFromBase64_Test.cs [hunk 1]
```diff
@@ -141,9 +141,9 @@ public void TestAnalyseFromBase64_nullBase64()
             {
                 var controller = new AnalyseController(_adaptationServiceClient, _storeConfiguration, _processingConfiguration, _analyseControllerLogger, _fileUtility, _zipUtility, _cloudSdkConfiguration, _httpService, _commonService);
                 Task<IActionResult> response = controller.AnalyseFromBase64(null);
-                Assert.AreNotEqual(response, null);
-                Assert.AreEqual(response.GetType().ToString(), "System.Threading.Tasks.Task`1[Microsoft.AspNetCore.Mvc.IActionResult]");
-                Assert.AreEqual(response.Result.ToString(), "Microsoft.AspNetCore.Mvc.BadRequestObjectResult");
+                Assert.AreNotEqual(null, response);
```
Comment (ggrig) [inline Source/Test/Glasswall.CS-SDK.UnitTests/AnalyseController_AnalyseFromBase64_Test.cs:144]: What is the reason for swapping the comparisons arguments here?

File: Source/Service/Controllers/FileTypeDetectionController.cs [hunk 2]
```diff
@@ -31,142 +31,17 @@ public class FileTypeDetectionController : CloudProxyController<FileTypeDetectio
         public async Task<IActionResult> DetermineFileTypeFromBase64([FromBody][Required] Base64Request request)
         {
             _logger.LogInformation($"[{_commonService.UserAgentInfo.ClientTypeString}]:: {nameof(DetermineFileTypeFromBase64)} method invoked");
+            return await ProcessFileTypeDetection(null, request, null, RequestType.Base64);
```
Comment (ggrig) [inline Source/Service/Controllers/FileTypeDetectionController.cs:34]: Where is ProcessFileTypeDetection defined?

File: Source/Test/Glasswall.CS-SDK.UnitTests/AnalyseController_AnalyseFromBase64_Test.cs (hunk 1 above)
Comment (mohitsengarr) [reply Source/Test/Glasswall.CS-SDK.UnitTests/AnalyseController_AnalyseFromBase64_Test.cs:144]: @ggrig  that is the correct way to use the Asserts. The first parameter is expected and second is the Actual value. Technically this is correct but fundamentally wrong. I don't want to change this but sonar check was failing due to this. Hence, I have made this change.

File: Source/Service/Controllers/FileTypeDetectionController.cs (hunk 2 above)
Comment (mohitsengarr) [reply Source/Service/Controllers/FileTypeDetectionController.cs:34]: Check the base controller.

File: Source/Test/Glasswall.CS-SDK.UnitTests/AnalyseController_AnalyseFromBase64_Test.cs (hunk 1 above)
Comment (ggrig) [reply Source/Test/Glasswall.CS-SDK.UnitTests/AnalyseController_AnalyseFromBase64_Test.cs:144]: Got it. Thanks

File: Source/Service/Controllers/FileTypeDetectionController.cs (hunk 2 above)
Comment (ggrig) [reply Source/Service/Controllers/FileTypeDetectionController.cs:34]: Yeap. Foud that

**Pull request merged**